src/hint_processor/builtin_hint_processor/uint256_utils.rs

```diff
@@ -171,6 +171,37 @@ pub fn uint256_unsigned_div_rem(
     vm: &mut VirtualMachine,
     ids_data: &HashMap<String, HintReference>,
     ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    uint256_offseted_unsigned_div_rem(vm, ids_data, ap_tracking, 0, 1)
+}
+
+/*
+Implements hint:
+%{
+    a = (ids.a.high << 128) + ids.a.low
+    div = (ids.div.b23 << 128) + ids.div.b01
+    quotient, remainder = divmod(a, div)
+
+    ids.quotient.low = quotient & ((1 << 128) - 1)
+    ids.quotient.high = quotient >> 128
+    ids.remainder.low = remainder & ((1 << 128) - 1)
+    ids.remainder.high = remainder >> 128
+%}
+*/
+pub fn uint256_expanded_unsigned_div_rem(
+    vm: &mut VirtualMachine,
+    ids_data: &HashMap<String, HintReference>,
+    ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    uint256_offseted_unsigned_div_rem(vm, ids_data, ap_tracking, 1, 3)
+}
+
+pub fn uint256_offseted_unsigned_div_rem(
+    vm: &mut VirtualMachine,
+    ids_data: &HashMap<String, HintReference>,
+    ap_tracking: &ApTracking,
+    offset_low: usize,
+    offset_high: usize,
```

Comment:
I would recommend for these to be called something like `div_offset_*`. Otherwise I need to read the body to understand what the offset applies to.

Reply:
Done!